1-data-marketplace/README.md

```diff
@@ -78,6 +76,14 @@ In order to run the automation for the Graph permissions follow the steps below:
 
       - GroupMember.Read.All
 
+
+After adding permissions to the Service Principal that was created, you simply need to grant administrative permissions using either App Registrations in the Azure Portal OR the Enterprise App in the Microsoft Enterprise Admin.
+
+Note: The new Service Principal can be found within the provision.config.file under "ama" -> "analytics" -> "identity".
```

Comment:
This note can now be deleted.
```suggestion
```

Reply:
do we have maybe a name for the user? I am afraid the person running this script would not know who to give admin permission. @ydaponte